Extension/test/unit/asyncIterators.test.ts

```diff
@@ -0,0 +1,51 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { describe, it } from 'mocha';
+import { fail, ok, strictEqual } from 'node:assert';
+import { accumulator } from '../../src/Utility/Async/iterators';
+
+// console.time();

```

Comment:
What are these two commented // console. lines for?

Reply:
leftover lines, removed.